channels/vn.m3u

```diff
@@ -1,33 +1,33 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="ANTV.vn" tvg-name="ANTV" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/antv.png" group-title="",ANTV
+#EXTINF:-1 tvg-id="antv.vn" tvg-name="ANTV" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://raw.githubusercontent.com/HaNoiIPTV/HaNoiIPTV.m3u/master/ANTV%20-%20Truy%E1%BB%81n%20h%C3%ACnh%20C%C3%B4ng%20an%20Nh%C3%A2n%20d%C3%A2n%20HD.png" group-title="",ANTV - Truyền hình An Ninh
 http://antvlive.ab5c6921.cdnviet.com/antv/chunklist.m3u8
-#EXTINF:-1 tvg-id="ANTVHD.vn" tvg-name="ANTV HD" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/antv.png" group-title="",ANTV HD (576p)
-http://antvlive.ab5c6921.cdnviet.com/antv/playlist.m3u8
-#EXTINF:-1 tvg-id="BRT.vn" tvg-name="BRT" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://brt.vn/common/v2/images/Banner150219.png" group-title="",BRT (360p)
+#EXTINF:-1 tvg-id="antv.vn" tvg-name="ANTV HD" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://raw.githubusercontent.com/HaNoiIPTV/HaNoiIPTV.m3u/master/ANTV%20-%20Truy%E1%BB%81n%20h%C3%ACnh%20C%C3%B4ng%20an%20Nh%C3%A2n%20d%C3%A2n%20HD.png" group-title="",ANTV - Truyền hình An Ninh HD (720p)
+http://gg.gg/HaNoiIPTV-ANTV-Truyen_hinh_An_Ninh_HD
```

Comment:
https://github.com/iptv-org/iptv/pull/2914#issuecomment-849584583

Reply:
Nothing wrong with http://antvlive.ab5c6921.cdnviet.com/antv/playlist.m3u8
New link (gg.gg) redirects to https://livecdn.fptplay.net/sdc/anninhtv_hls.smil/chunklist_b2500000.m3u8

Reply:
Not really. Both links do work in SimpleTV